tests/utils.py

```diff
@@ -256,19 +256,23 @@ def assert_guest_os_memory_amount(vm, spec_memory_amount):
     )
 
 
-def assert_restart_required_condition(vm, expected_message):
-    def _get_restart_required_condition(vm):
-        return [condition for condition in vm.instance.status.conditions if condition.type == "RestartRequired"][0]
+def get_restart_required_condition(vm):
```

Comment:
this function is not needed, it just wraps `get_condition_message`, please use directly in the sampler

Reply:
I believe there was an intention to reuse this function for [this PR](https://github.com/RedHatQE/openshift-virtualization-tests/pull/1659), but that depends on @Anatw, and it’s possible that it’s no longer relevant.

Reply:
it is just a wrapper; where needed, call `get_condition_message`